### PR TITLE
feat: Upgrade tree to v3

### PR DESCRIPTION
**Pull request by GordonSmith**
## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [x] I have raised new issues to address them separately

### CODE REVIEW

File: packages/tree/tests/tree.browser.spec.ts [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+                                break;
+                            case Treemap:
+                                render(new Treemap()
+                                    .columns(["label", "size"])
+                                    .data(flare.data)
+                                );
+                                break;
+
+                            default:
+                                it("Has render test", () => {
```
Comment (jeclrsg) [inline packages/tree/tests/tree.browser.spec.ts:440]: I believe it's been the case in the last few PRs I reviewed also, but I only just thought about it now. Should these other cases not also have some kind of `it(...)` expectation in them as well?

**Review by jeclrsg — APPROVED** [review]
@GordonSmith just one question about the tests, approved though

File: packages/tree/tests/tree.browser.spec.ts (hunk 1 above)
Comment (GordonSmith) [reply packages/tree/tests/tree.browser.spec.ts:440]: The `it` call is buried inside the "render" function, it does a few checks but the main one is:
1. Set target to a placeholder div
2. Renders the widget
3. Sets the target to null
4. Sets the target to a new placeholder
5. Renders the widget

Comment (jeclrsg) [reply packages/tree/tests/tree.browser.spec.ts:440]: I realized that when I was just reviewing another; chased what that `render(...)` import actually was

**Pull request merged**